src/azul/strings.py

```diff
@@ -223,3 +227,58 @@ def longest_common_prefix(strings: Iterable[str]) -> Optional[str]:
         if s2[i] != c:
             return s1[:i]
     return s1
+
+
+def jl(*lines: str) -> str:
+    """
+    Return a newline-separated string.
+    """
+    return '\n'.join(lines)
+
+
+def jw(*words: str) -> str:
+    """
+    Return a space-separated string.
+    """
+    return ' '.join(words)
+
+
+def surround(string: str, end: str) -> str:
+    """
+    Prefix and postfix a string with another. The pre/postfix cannot be a
+    substring of the source string.
```

Comment:
Nitpick, but I think the English can be streamlined a bit
```suggestion
    Prepend and append an affix to a string. The affix cannot be a
    substring of the base string.
```

Reply:
(feel free to disregard if you feel "affix" is too obscure a word to be helpful here)